po/sv.po

```diff
@@ -0,0 +1,249 @@
+# Russian translation for Ladybird.
+msgid ""
+msgstr ""
+"Project-Id-Version: ladybird wip\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2023-08-29 11:05+0300\n"
+"Language: sv\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=n != 1;\n"
+
+#. Translators: this is a format string for the zoom-percent label in the main menu.
+#. For most languages, it doesn't need translating.
+#: Window.cpp:353
+#, c-format
+msgid "%d%%"
+msgstr ""
+
+#. Translators: this is a placeholder content for the Bookmarks & History
+#. popover; it will go away once we implement it, so you can skip translating
+#. it. But it also doesn't hurt to translate it either, if you want to.
+#: window.blp:51
+msgid "...will appear here, once we implement them :^)"
+msgstr "...kommer visas här, när de är implementerade :^)"
+
+#: window.blp:143 window.blp:242
+msgid "Bookmarks"
+msgstr "Bokmärken"
+
+#: window.blp:47
+#, fuzzy
+msgid "Bookmarks & History"
+msgstr "Bokmärken & Historik"
+
+#: window.blp:33
+msgid "Close _Other Tabs"
+msgstr "Stäng _Övriga Flikar"
```

Comment:
Question: do you also use Header Capitalization in Swedish, where basically every word is capitalized? Or is that an English thing? In other words, are you doing this intentionally or unintentionally here?

Reply:
> Question: do you also use Header Capitalization in Swedish, where basically every word is capitalized? Or is that an English thing? In other words, are you doing this intentionally or unintentionally here?

I just assumed it was a design pattern based on it is look weird in English. Will correct it right away! =)

> Another question: is the Ö symbol easily to type? (i.e. does it typing it require a key combination?) Because if it's non-trivial to type, you should choose another accelerator key for this item.

Yes, we have dedicated keys for å, ä and ö on Swedish and Nordic keyboard layouts and that's basically on every keyboard sold in Sweden.